README.md

```diff
@@ -97,6 +98,17 @@ export VK_COMPILE_TIME_LOG=<log-file-path>
 
 To check if the layers are enabled, you can run a sample Vulkan application (such as `vkcube`) and look for the generated layer log files, or check the [Vulkan loader logs](https://github.com/KhronosGroup/Vulkan-Loader/blob/master/docs/LoaderInterfaceArchitecture.md#table-of-debug-environment-variables).
 
+### Log formats:
+
+To visualize the logs, we use multiple trace viewers such as the python scripts in the `scripts/` directory and Perfetto (https://ui.perfetto.dev/). To suit the input format requirements of each logger, we support logging in various formats such as CSV and Chrome Trace Event. see:
+https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview
+
+Our logging system also supports custom formats (e.g., `sample_output/events.log`).
+
+#### Adding a new log format:
+
+To add a custom format, create a new logger by extending `EventLogger` class, the base class for all the loggers. Then, introduce the new logger to the `LayerData` through its constructor.
```

Comment:
This first developer documentation more than the user-facing one. I'd leave this out.